.ruby-version

```diff
@@ -1 +1 @@
-jruby-9.1.12.0
+jruby-9.1.13.0
```

Comment:
We tried this before on https://github.com/elastic/logstash/pull/8219 , but it caused issues so we reverted it: https://github.com/elastic/logstash/pull/8219

The Jenkins issues should be resolved now via other channels, but it can still break local devs ability to bootstrap.

Reply:
will revert